aerogear/unifiedpush-operator
# WebPush support

## Motivation
WebPush support in UPS requires additional queues for sending messages.  This PR creates those queues.

## What
I've added two new queues to the list for the operator to manager, `WebPushMessageQueue` and `WebTokenBatchQueue`.  

## Verification Steps
 
1. Run the operator and deploy UPS normally
2. Verify the new queues are created.
